docs/usage/types/callables.md

```diff
@@ -2,6 +2,9 @@
 description: Support for callable types.
 ---
 
+`typing.Callable`
+: see below for more detail on parsing and validation
```

Comment:
Another example

Reply:
These are artifacts from the old docs that need to get fixed in a separate editorial pass. In this pass I was just trying to get all the correct page content set up.